index.html

```diff
@@ -219,13 +225,13 @@ <h3>legend</h3>
                       bugArray = [bugArray];
                     }
                     for (var bug of bugArray) {
-                      out += `<a href='${bug}' title='mozilla bug'><span class='glyphicon glyphicon-link' aria-hidden='true'></span><span class='sr-only'>mozilla bug</span></a> `;
+                      out += iconLink(bug, 'mozilla bug', 'wrench');
                     }
                   }
                   if (row.ciuName && row.ciuName !== "") {
-                    out += "<a href='https://caniuse.com/#feat=" + row.ciuName + "' title='caniuse'><span class='glyphicon glyphicon-eye-open' aria-hidden='true'></span><span class='sr-only'>caniuse</span></a> "
+                    out += iconLink(`https://caniuse.com/#feat=${row.ciuName}`, 'caniuse', 'question-sign');
```

Comment:
I guess I think neither of these [icons](https://getbootstrap.com/docs/3.3/components/#glyphicons) (`eye-open` or `question-sign`) are particularly informative -- perhaps `list` or `tasks` might be better?